packages/ag-charts-community/src/chart/mapping/prepare.ts

```diff
@@ -176,6 +176,47 @@ function sanityCheckOptions<T extends AgChartOptions>(options: T) {
     });
 }
 
+function validateSoloSeries<T extends AgChartOptions>(options: T): T {
+    function isSoloSeries(series: SeriesOptionsTypes) {
+        return series.type === 'bullet';
+    }
+    function hasSoloSeries(options: SeriesOptionsTypes[]) {
+        for (const series of options) {
+            if (isSoloSeries(series)) return true;
+        }
+        return false;
+    }
```

Comment:
We should probably encode this as part of `SeriesModule` to avoid having a series-specific behaviour.

Reply:
Good suggestion. I'll change this.